lt/client/client.go

```diff
@@ -291,17 +301,23 @@ func (u *User) transmitSpeech() {
 		log.Fatalf("%s: failed to create opus encoder: %s", u.cfg.Username, err.Error())
 	}
 
-	if err := u.callsClient.Unmute(track); err != nil {
-		log.Fatalf(err.Error())
-	}
-
 	for text := range u.speechTextCh {
 		func() {
 			defer func() {
+				time.Sleep(100 * time.Millisecond)
+				if err := u.Mute(); err != nil {
+					log.Fatalf(err.Error())
+				}
+				log.Printf("muted")
```

Comment:
If we need to log we should include the username. Same for the unmuted case below.

Reply:
Totally, thanks.